docusaurus.config.js

```diff
@@ -80,6 +80,15 @@ const config = {
     ],
   ],
 
+  plugins: [
+    [
+      '@docusaurus/plugin-google-gtag',
+      {
+        trackingID: 'G-G856J3S8ZG',
+        anonymizeID: true,
+      },
+    ],
+  ],
```

Comment:
We are using the @docusaurus/preset-classic so I think you just follow the instructions for the preset options
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag

Reply:
Ya I do think as if we are using `@docusaurus/preset-classic ` and they mentioned that there is no need to install this plugin as a dependency, then we can skip this step and use it as a preset option as mentioned in the documentation

Reply:
Hi, yeah I read about the implementation for `@docusaurus/preset-classic `. However, I was a bit confused between `@docusaurus/preset-classic` and just `classic` on line 46 if they were the same or not. So I used plugin for safety. I will now implement as per `@docusaurus/preset-classic `. Should I also change `classic` to `@docusaurus/preset-classic ` on line 46?